ibu-imager/cmd/create.go

```diff
@@ -79,9 +80,14 @@ func create() {
 
 	// Check if containerRegistry was provided by the user
 	if containerRegistry == "" {
-		fmt.Printf(" *** Please provide a valid container registry to store the created OCI images *** \n")
-		log.Info("Skipping OCI image creation.")
-		return
+		log.Printf(" *** Please provide a valid container registry to store the created OCI images *** ")
+		log.Fatal("Skipping OCI image creation.")
+	}
+
+	// Check if recertContainerImage was provided by the user
+	if recertContainerImage == "" {
```

Comment:
I believe we can just use https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired